src/BloomBrowserUI/bookEdit/BookPagesThumbnailList/BookPagesThumbnailList.htm

```diff
@@ -32,8 +33,11 @@
                 fireCSharpEvent("gridClick", $(this).attr('id'));
             });
 
-			// The 100 ms value is based on experimentation. It may need to be re-evaluated in the future.
-            setTimeout(loadNextThumbnail, 100);
+			// start the thumbnail timer
+            var timerSetting = document.getElementById('thumbnailInterval');
+            if (timerSetting)
+                thumbnailTimerInterval = parseInt(timerSetting.value);
+            setTimeout(loadNextThumbnail, thumbnailTimerInterval * 2);
```

Comment:
I'm not seeing a reason for the *2, so a comment explaining it is probably appropriate.